test/extension.singleline.test.ts

```diff
@@ -3,7 +3,7 @@ import { runTest as Test, runTestQuickPick as QuickPick } from './runner'
 
 const config = vsc.workspace.getConfiguration('postfix')
 
-describe('Single line template tests', () => {
+describe.only('Single line template tests', () => {
```

Comment:
Don't forget to remove this `.only` ;)

Reply:
Oh, it was actually my fault... when you refresh your branch from latest `develop` this should disappear.